lib/pluginmanager/templates/codec-plugin/lib/logstash/codecs/example.rb.erb

```diff
@@ -25,6 +25,7 @@ class LogStash::Codecs::<%= classify(plugin_name) %> < LogStash::Codecs::Base
   config :append, :validate => :string, :default => ', Hello World!'
 
   def register
+    require "logstash/codecs/line"
     @lines = LogStash::Codecs::Line.new
```

Comment:
Thanks for noticing this.
Actually we can rely on LogStash::Plugin.lookup so that the requires are resolved for us, so I suggest:

```suggestion
    @lines = LogStash::Plugin.lookup("codec", "line").new
```